stdlib/public/Darwin/Accelerate/CMakeLists.txt

```diff
@@ -2,19 +2,22 @@ cmake_minimum_required(VERSION 3.4.3)
 include("../../../../cmake/modules/StandaloneOverlay.cmake")
 
 add_swift_target_library(swiftAccelerate ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
-  BNNS.swift.gyb
+BNNS.swift.gyb
+Quadrature.swift
```

Comment:
Can you revert the whitespace changes here? They add a little bit of noise to the simplest application of git blame if we're trying to track down a problem in the future. It looks like there should only be a single line change in this file.